<ui version="4.0" >
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Logon to Catalog</string>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="okBox" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>93</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>8</x>
     <y>9</y>
     <width>191</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Federal Id:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="fedIdEdit" >
      <property name="toolTip" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="passwordEdit" >
      <property name="toolTip" >
       <string/>
      </property>
      <property name="echoMode" >
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="helpButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>99</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Help For ICat Login Algorithm</string>
   </property>
   <property name="text" >
    <string>?</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fedIdEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>okBox</tabstop>
  <tabstop>helpButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okBox</sender>
   <signal>accepted()</signal>
   <receiver>LoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
